notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -1126,6 +1126,73 @@ the art of entrepreneurship [The hardened businessman Peter Drummond laughed of
 human self-sacrifice [Bug gave his life to save those of his friends in an ultimately unsuccessful to destroy the video game monster run amok.]
 
 
+movie: Jimmy Neutron: Boy Genius (2001)
+=======================================
+
+:: Title
+Jimmy Neutron: Boy Genius
+
+:: Description
+Jimmy Neutron: Boy Genius is a 2001 American computer-animated science fiction
+comedy film produced by Nickelodeon Movies, O Entertainment and DNA
+Productions, and distributed by Paramount Pictures.
+
+Synopsis: A schoolboy with super-genius intelligence must save all of the
+parents of the world from a race of egg-like aliens known as the Yolkians.
+
+:: Date
+2001-12-21
+
+:: References
+https://en.wikipedia.org/wiki/Jimmy_Neutron:_Boy_Genius
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+extraterrestrial being [The egg-like Yolkian aliens abducted all the adults in Retroville with the intention of sacrificing them to a gigantic three-eyed alien chicken.]
```

Comment:
Egg-like huh? I suppose that's technically https://www.themeontology.org/theme.php?name=animal%20and%20plant%20patterned%20being no?